docs/index.rst

```diff
@@ -57,12 +59,16 @@ Example Application
           description: Get a random pet
           responses:
             200:
+              description: Return a pet
               content:
                 application/json:
                   schema: PetSchema
         """
-        pet = get_random_pet()
-        return PetSchema().dump(pet)
+        pet_data = {
+            "name": "sample_pet_" + str(uuid.uuid1()),
+            "categories": [{"id": 1, "name": "sample_category"}],
+        }
+        return PetSchema().dump(hardcoded_pet)
```

Comment:
Oops. Inconsistency.

Proposal:

```py
        # Hardcoded example data
        pet_data = {
            "name": "sample_pet_" + str(uuid.uuid1()),
            "categories": [{"id": 1, "name": "sample_category"}],
        }
        return PetSchema().dump(pet_data)
```

Reply:
damn... sorry for that. It should be fine now, I've re-tested it to make sure its running file